app/screens/home/search/modifiers/show_more.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import FormattedText from '@components/formatted_text';
+import MenuItem from '@components/menu_item';
+import {useTheme} from '@context/theme';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type ShowMoreButtonProps = {
+    onPress: () => void;
+    showMore: boolean;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        showMore: {
+            color: theme.buttonBg,
+            paddingLeft: 20,
+            paddingVertical: 12,
```

Comment:
I think you can remove the vertical padding. It seems to inherit the correct padding already without this